web/client/epics/catalog.js

```diff
@@ -290,6 +290,40 @@ export default (API) => ({
                         .merge(Rx.Observable.from(actions))
                         .catch((e) => Rx.Observable.of(describeError(layer, e)));
                 }
+                if (layer.type === 'model') {
+                    const properties = layer?.features?.[0]?.properties || {};
+                    if (properties?.projectedCrsNotSupported || !properties?.projectedCrs) {
+                        const { center: mapCenter } = mapSelector(state) || {};
+                        const center = CoordinatesUtils.reproject(mapCenter, mapCenter.crs, 'EPSG:4326');
+                        const longitude = center.x;
+                        const latitude = center.y;
+                        const newLayer = {
+                            ...layer,
+                            bbox: {
+                                bounds: {
+                                    minx: longitude || 0 - 0.001,
+                                    miny: latitude || 0 - 0.001,
+                                    maxx: longitude || 0 + 0.001,
+                                    maxy: latitude || 0 + 0.001
+                                },
+                                crs: 'EPSG:4326'
+                            },
+                            features: [
+                                {
+                                    ...layer?.features?.[0],
+                                    geometry: {
+                                        type: 'Point',
+                                        coordinates: [longitude, latitude, 0]
+                                    }
+                                }
+                            ]
+                        };
+                        return Rx.Observable.of(
+                            // add notification warning,
```

Comment:
Here we introduced some logic to place the model on the center of the current view if the projected crs is not Supported or the projectedCrs is undefined. We need to add a notification warning action that explain to the user the following:

- the model is placed on the center of the current view because: 
  - in case of projectedCrs undefined: there are not information related to the projection
  - in case of projectedCrsNotSupported: there are information related to the projection but the crs is not supported

Please create user friendly message test based on these cases